### PR TITLE
12078 get object keys

### PR DESCRIPTION
**Pull request by dumagalex**
Enhanced get keys test coverage (with wrapper test flow)
Removed local test suite which remained for validate json's tests
Renamed conflicting name of compare_lists.sl test flow to test_compare_lists.sl

### CODE REVIEW

File: test/io/cloudslang/base/json/test_get_keys.sl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+flow:
+  name: test_get_keys
+
+  inputs:
+    - json_input
+    - json_path
+    - expected_keys
+
+  workflow:
+    - getKeys:
```
Comment (Bonczidai) [inline test/io/cloudslang/base/json/test_get_keys.sl]: IMHO snake case

File: test/io/cloudslang/base/json/test_get_keys.sl [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        do:
+          get_keys:
+            - json_input
+            - json_path
+
```
Comment (Bonczidai) [inline test/io/cloudslang/base/json/test_get_keys.sl]: Y the newlines between the same elements of a step?

Comment (dumagalex) [inline test/io/cloudslang/base/json/test_get_keys.sl]: You mean the newlines between do and publish? I though the code is easier to read with spacing. I can remove them if needed

Comment (Bonczidai) [inline test/io/cloudslang/base/json/test_get_keys.sl]: I can leave with it

File: test/io/cloudslang/base/json/test_get_keys.sl (hunk 1 above)
Comment (Bonczidai) [inline test/io/cloudslang/base/json/test_get_keys.sl]: @dumagalex how about this one?

Comment (dumagalex) [inline test/io/cloudslang/base/json/test_get_keys.sl]: changing it to snake case

**Pull request merged**